rmgpy/reaction.py

```diff
@@ -1321,28 +1326,33 @@ def get_mean_sigma_and_epsilon(self, reverse=False):
         mean_epsilons = reduce((lambda x, y: x * y), epsilons) ** (1 / len(epsilons))
         return mean_sigmas, mean_epsilons
 
-def isomorphic_species_lists(list1, list2, check_identical=False, only_check_label=False, generateInitialMap=False):
+
+def same_species_lists(list1, list2, check_identical=False, only_check_label=False, generate_initial_map=False, strict=True):
```

Comment:
If we're renaming this already can we either generalize it so it works for N species or rename it in a less general way?

Reply:
You mean since it can only compare list of up to 3 species? Do you any name suggestions?

Reply:
isomorphic_species_trios?

Reply:
isomorphic_species_lists_le3, isomorphic_species_lists_lt4?

Reply:
Do you think it's not worth worrying about the confusion that much?